### PR TITLE
Revamp view order and quote pages

### PR DESCRIPTION
**Pull request by matt-downs**
- Only show pay now button on view order screen if there is an amount owed #338
- Fix issue where pay now button would be displayed once for each orderline in nr view order
- Duplicate view order template to nr view order template to remove legacy code
- Duplicate view quote template to nr view quote template to remove legacy code

### CODE REVIEW

File: src/templates/customer/nr_view_order/order.template.html [hunk 1]
```diff
@@ -1,39 +1,27 @@
...
+				[%/param%]
+			[%/show_order%]
+		</div>
+	</div>
+	[%print_order id:'[@order_id@]'/%]
 </div>
-
-<script language="javascript" type="text/javascript">
-function print_invoice() {
-	var win = window.open('[%URL page:"account" type:"nr_print" fn:"invoice" id:"[@order_id@]" qs:"code=[@code@]"%][%END URL%]', 'print_invoice', 'width=740,height=400,scrollbars=yes,resizeable=yes');
```
Comment (matt-downs) [inline src/templates/customer/nr_view_order/order.template.html]: Might need to check if `qs:"code=[@code@]"` is important.

It's not in the nr_view_order/quote template though...

File: src/templates/customer/nr_view_order/quote.template.html [hunk 2]
```diff
@@ -1,36 +1,23 @@
...
+		<h1>View Quote</h1>
+	</div>
+	<script language="javascript" type="text/javascript">
+		function print_quote() {
+			var win = window.open('[%url page:'account' type:'print' fn:'invoice' id:'[@order_id@]'/%]', 'print_invoice', 'width=740,height=400,scrollbars=yes,resizeable=yes');
+		}
+	</script>
+	<div class="row">
+		<div class="col-xs-12">
+			<input type="button" class="btn btn-default pull-left" value="Print Quote" onClick="print_quote()">&nbsp;
```
Comment (philcon93) [inline src/templates/customer/nr_view_order/quote.template.html]: Place icon back in please - <i class="fa fa-check icon-white"></i>

Comment (philcon93) [reply src/templates/customer/nr_view_order/quote.template.html]: Actually don't worry, if it's not in the invoice one then let's keep it consistent.

Comment (matt-downs) [reply src/templates/customer/nr_view_order/quote.template.html]: @philcon93 actually you raise a good point though... Should the print button have a printer icon? 
image

The button on the my account page does:
image

**Pull request merged**